### PR TITLE
Flash KeyValuePage item when callback

### PR DESCRIPTION
**Pull request by robert00s**
Similar flash like tap on button or menu item.

I'm not sure that this is too expensive:
`UIManager:setDirty("all", 'ui')`
pinging @poire-z for help :)

### CODE REVIEW

File: frontend/ui/widget/keyvaluepage.lua [hunk 1]
```diff
@@ -210,7 +210,17 @@ function KeyValueItem:init()
 end
 
 function KeyValueItem:onTap()
-    self.callback()
+    if self.callback then
+        UIManager:scheduleIn(0.0, function()
```
Comment (Frenzie) [inline frontend/ui/widget/keyvaluepage.lua]: Is that any different than nextTick?

**Comment by poire-z** [comment]
> I'm not sure that this is too expensive:

It probably is, guess it's doing a full refresh (but a "ui" one, but still).
Usually, small widgets (like iconbutton.lua, from which i took this) do:
```lua
        UIManager:setDirty(self.show_parent, function()
            return "ui", self[1].dimen
        end)
```
Not sure why it should pass the parent, you could try with `self` instead of `self.show_parent`.
Or with `self[1].dimen` or `self.dimen` (I once really looked at how all that worked, but I forgot, nowadays I just try things...)
If it doesn't work, you would need in KeyValuePage:_populateItems(), to add it to`KeyValueItem:new{ ... , show_parent = self...}`

**Comment by robert00s** [comment]
@poire-z 

> If it doesn't work, you would need in KeyValuePage:_populateItems(), to add it toKeyValueItem:new{ ... , show_parent = self...}

This is what I need. Thanks.

@Frenzie 
I can use that code but I can see difference in usage (the same visual experiance)

```
    if self.callback then
            self[1].invert = true
            UIManager:setDirty(self.show_parent, function()
                return "ui", self[1].dimen
            end)
        UIManager:forceRePaint()
        UIManager:nextTick(function()
            self.callback()
            self[1].invert = false
            UIManager:setDirty(self.show_parent, function()
                return "ui", self[1].dimen
            end)
        end)
    end
    return true

```

**Comment by Frenzie** [comment]
I only meant that in place of `UIManager:scheduleIn(0.0, function()`, 0.1 is a minor timeout instead.

**Pull request merged**